volatility3/framework/plugins/windows/pstree.py

```diff
@@ -132,6 +138,23 @@ def yield_processes(pid, descendant: bool = False):
                 proc.get_exit_time(),
             )
 
+            if self.config["verbose"]:
+                row += (
+                    proc.SeAuditProcessCreationInfo.ImageFileName.Name.get_string()
```

Comment:
What happens if this throws an InvalidAddressExcption when try to access this?  At the moment everything will stop, would it be better to catch it as just below?

Reply:
Added a catch for SeAuditProcessCreationInfo throwing an exception with [97ea676](https://github.com/volatilityfoundation/volatility3/pull/1076/commits/97ea6765ea57df54e3a6721c86b00c6ebe7084d9). There are also some cases where `SeAuditProcessCreationInfo.ImageFileName.Name` is able to be accessed, but returns the empty string. I kept the logic for checking this case and added a comment